en/SetupView/Motors.md

```diff
@@ -1,23 +1,30 @@
 # Motors Setup
 
-This section is used to setup and test individual motors/servos, and applies to both ArduPilot and PX4:
+Motor Setup is used to test individual motors/servos (for example, to verify that motors spin in the correct direction).
 
-1. Remove all propellers before running the test.
-   > **Warning** You must do this before activating the motors (next step)
-1. Slide the switch to enable motor sliders (labeled "Propellers are removed - Enable motor sliders").
+> **Tip** These instructions apply to PX4 and to most vehicle types on ArduPilot.
+  Vehicle-specific instructions are provided as sub-topics (e.g. [Motors Setup (ArduSub)](../SetupView/Motors_ardusub.md)).
+
+![Motors Test](../../assets/setup/Motors.png)
+
+## Test Steps
+
+
+To test the motors:
+
+1. Remove all propellers.
+   > **Warning** You must remove props before activating the motors!
+1. (*PX4-only*) Enable safety switch - if used.
+1. Slide the switch to enable motor sliders (labeled: *Propellers are removed - Enable motor sliders*).
 1. Adjust the individual sliders to spin the motors and confirm they spin in the correct direction.
    > **Note** The motors only spin after you release the slider and will automatically stop spinning after 3 seconds.
 
-<!-- PX4 Firmware specific:
-- If a safety button is used, it must be pressed before motor testing is allowed.
-- The kill-switch still works to stop motors immediately.
-- The parameter `COM_MOT_TEST_EN` can be used to completely disable motor testing.
-- On boards with an IO, only the MAIN pins can be tested.
-- On the shell, `motor_test` can be used as well, which has additional options.
--->
-
-![Motors Test](../../assets/setup/Motors.png)
+## PX4 Notes
 
-Vehicle specific instructions are provided in the following topics:
-* [Motors Setup (ArduSub)](../SetupView/Motors_ardusub.md)
+The following notes apply PX4 only:
```

Comment:
I perfer to keep these within the Firmware docs, as the behavior might change with Firmware updates.

Reply:
@bkueng I removed this and merged. I see the argument but not completely convinced. In any case,  I am assuming you are adding docs for PX4.